lib/sql.js

```diff
@@ -75,7 +75,7 @@ SQLConnector.prototype.autoupdate = function(models, cb) {
 
   models = models || Object.keys(this._models);
 
-  async.eachSeries(models, function(model, done) {
+  async.each(models, function(model, done) {
```

Comment:
Please revert this change:

1. It looks unrelated to your fix.
2. It introduces a potential race condition, where models are created in different order each time `autoupdate` is executed.

Reply:
The commit message mentions that you are trying to fix a different bug using this commit. Feel free to open a new pull request to discuss this change and how to fix that other bug.

Reply:
Agreed.  I was having problems with this fix but have created a workaround.